### PR TITLE
Chore: Pass getUserProfileUrl() in contentSidebarProps in preview.html

### PR DESCRIPTION
**Pull request by pramodsum**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @pramodsum has signed the CLA. Thanks for the pull request!

File: src/components/ContentPreview/ContentPreview.js [hunk 1]
```diff
@@ -73,7 +73,8 @@ type Props = {
     onMetric?: Function,
     requestInterceptor?: Function,
     responseInterceptor?: Function,
-    previewInstance?: any
+    previewInstance?: any,
+    getUserProfileUrl?: Function
```
Comment (jeremypress) [inline src/components/ContentPreview/ContentPreview.js]: Shouldn't this be in `contentSidebarProps`?

Comment (JustinHoldstock) [inline src/components/ContentPreview/ContentPreview.js]: Could you create a more descriptive type for this? 

IE) that it returns a promise that resolves in a string?

Comment (pramodsum) [reply src/components/ContentPreview/ContentPreview.js]: Good point. Testing that out now

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #361

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: src/components/ContentPreview/ContentPreview.js (hunk 1 above)
Comment (pramodsum) [reply src/components/ContentPreview/ContentPreview.js]: @jeremypress fixed it

**Pull request merged**